frontend/src/pages/AccountSend.vue

```diff
@@ -304,21 +319,16 @@ function useSendForm() {
   }
 
   const isNativeToken = (address: string) => getAddress(address) === NATIVE_TOKEN.value.address;
-  const getMinSendAmount = (tokenAddress: string) => {
-    const chainId = BigNumber.from(currentChain.value?.chainId).toNumber();
-    // Polygon
-    if (chainId === 137) {
-      if (isNativeToken(tokenAddress)) {
-        return 1.0;
-      } else if (tokenAddress === '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619') {
-        return 0.001; // weth token minimum
-      } else {
-        return 3; // stablecoin token minimum
-      }
-    }
-    // Mainnet, Rinkeby, and other networks have higher ETH and stablecoin minimums due to higher fees
-    if (isNativeToken(tokenAddress)) return 0.01;
-    else return 100; // stablecoin token minimum
+
+  const getMinSendAmount = (tokenAddress: string): number => {
+    const tokenInfo = tokenList.value.filter((token) => token.address === tokenAddress)[0];
+    if (!tokenInfo) return 0;
```

Comment:
I agree. I had initially taken that approach, but then that led to noisy errors popping up when switching chains (I believe) because there was a race condition between the provider returning chain data and the backend returning token data. That might not be a problem anymore though. Let me double check...

Reply:
It's no longer a problem. Removed 👍